src/librustc_mir/util/liveness.rs

```diff
@@ -110,19 +110,26 @@ impl LivenessResults {
             ),
         }
     }
+
+    pub fn make_simulate_buffer<'tcx>(&self, mir: &Mir<'tcx>) -> LocalSet {
```

Comment:
can you add a comment here about how this buffer is held across invokes of `simulate_block` for performance reasons?